bco_api/api/migrations/0001_initial.py

```diff
@@ -1,5 +1,3 @@
-# Generated by Django 3.2 on 2021-10-01 12:33
```

Comment:
@chrisarmstrong151 you need to incorporate the existing migrations for this branch